tools/autodeployment/slack_app/index.js

```diff
@@ -0,0 +1,54 @@
+require('dotenv').config();
+const axios = require('axios').default;
+
+const { SLACK_MESSAGE_HOOK } = process.env;
+
+const prodStatus = () => {
+  let alarm = false;
+
+  setInterval(async () => {
```

Comment:
If we end up keeping this, I think it's better to use [setInterval Async](https://www.npmjs.com/package/set-interval-async), like we do in the [wiki-feed-parser](https://github.com/Seneca-CDOT/telescope/blob/38be67c1c0692df619e21174f3657eb661a47464/src/backend/utils/wiki-feed-parser.js#L4).